client/src/components/WarmFuzziesIndex.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
+import WarmFuzzyClient from "../apiClient/WarmFuzzyClient";
+import WarmFuzzyItem from "../components/WarmFuzzyItem";
+import { useAuthentication } from "../providers/Authentication";
+
+const WarmFuzziesIndex = (props) => {
+  const { user } = useAuthentication();
+  const [warmFuzzies, setWarmFuzzies] = useState([]);
+
+  const fetchWarmFuzzies = () => {
+    const client = new WarmFuzzyClient()
+    const allWarmFuzzies = client.getAllFuzzies()
+      .then(resp => {
+        return resp;
+      })
+      .then(fuzzies => {
+        console.log("returned fuzzies here")
+        setWarmFuzzies(fuzzies)
+      })
+  }
+
+  useEffect(fetchWarmFuzzies, []);
+
+  const allFuzzies = warmFuzzies.map((fuzzy => {
+    return (
+      <WarmFuzzyItem
```

Comment:
we should send the whole `WarmFuzzy` as a prop to this component.

```jsx
<WarmFuzzyItem
  warmFuzzy={fuzzy} />
```

Reply:
Refactored in the most recent PR